### ISSUE TITLE
webview_flutter is not opening download links on android. 

### ISSUE THREAD

**Issue by autumnlewjb** [flutter/flutter#62874]
I've tried to click on a  download link on my android device in debugging mode. However, nothing happens. Is there any possible solution for this? 
I'm currently using `webview_flutter: ^0.3.22+1` . Tried the `webview_flutter: ^0.3.22` but still the same.

**Comment by autumnlewjb** [comment]
I found a similar closed issue but it seems like the PR is for iOS and not android. Correct me if I'm wrong.

**Comment by darshankawar** [comment]
Hi @autumnlewjb,
Please provide `flutter doctor -v` and a minimal complete reproducible code sample that shows the issue you are facing.
Thanks.

**Comment by autumnlewjb** [comment]
> Please provide `flutter doctor -v` and a minimal complete reproducible code sample that shows the issue you are facing.

While trying to make a reproducible code as requested, I noticed that I was able to open some links with target=_"_blank"_ attribute. After some inspections, I found out that my issue is actually as described in the issue #39321 - problem opening download links. 

Since the issue I'm encountering is posted and still opening at the time being, I think it's better to close this issue after mentioning the similar issue as mine. 

A big thanks to @darshankawar for corresponding.
